src/components/Flexbetween.jsx

```diff
@@ -0,0 +1,10 @@
+import { Box } from "@mui/material";
```

Comment:
was this component required? If so, it should be stored in reusable components

Reply:
Instead of writing all time flex box style, I created this.

Reply:
> Instead of writing all time flex box style, I created this.

yes, then that's a reusable component. Move that to reusable components folder